packages/expo-cli/src/credentials/api/AndroidApi.ts

```diff
@@ -31,7 +32,15 @@ export default class AndroidApi {
     return this.credentials[experienceName];
   }
 
-  public async updateKeystore(experienceName: string, keystore: Keystore): Promise<void> {
+  public async updateKeystore(
```

Comment:
This is a wrapper for api calls only, there should not be any logic unrelated to that here. Especially because that code implements local caching of credentials it's potentially easy to break and introduce very hard to find bugs.